3.resistance-signature/scripts/nbconverted/8.1. last-batch-compile-otherclone-datase.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
```

Comment:
there are two typos in the file name (a space and `datase` instead of `dataset` i presume)